### ISSUE TITLE
Atlas minimap covers potion effect indicators in top right corner

### ISSUE THREAD

**Issue by DefinitelyNotFred** [Hunternif/AntiqueAtlas#231]
Ideally those indicators would be beside the book somewhere instead of hidden by it. I'm using a mod that hides the swirling potion effects on the screen so it's nice to have those indicators there to see at a glance that i did indeed get hit and to see what potions I got hit with.

**Comment by tyra314** [comment]
In 1.12.2 and older you can configure the position of the minimap. This will return for the 1.14-fabric version, once the config is done.
